### PR TITLE
[DONE] Timeseries redesign

### PR DESCRIPTION
**Pull request by gijs**
screen shot 2016-12-06 at 12 46 45

### CODE REVIEW

**Comment by gijs** [comment]
Some proof of improvements:

screen shot 2016-12-07 at 11 46 59

screen shot 2016-12-07 at 11 50 05

screen shot 2016-12-07 at 11 51 54

File: app/components/dashboard/dashboard.html [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <div class="dashboard-wrapper drag-target">
 
+  <div ng-if="dashboard.graphs.length < 1" class="dashboard-empty-drag-target">
+    <p><strong translate>Zoek, sleep of selecteer een tijdreeks</strong></p>
```
Comment (ernstkui) [inline app/components/dashboard/dashboard.html]: The default should be english

File: app/components/omnibox/directives/omnibox-directives.js [hunk 2]
```diff
@@ -18,6 +18,10 @@ angular.module("omnibox")
         RETURNKEY: 13,
       };
 
+      scope.showOmnibox = function() {
```
Comment (ernstkui) [inline app/components/omnibox/directives/omnibox-directives.js]: What is this?

File: app/components/omnibox/templates/omnibox.html [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
   <search></search>
+
+
+
   <search-results ng-if="omnibox.searchResults"></search-results>
 
-  <div id="cards" class="pullDown cardbox">
+  <div id="cards"
+       class="pullDown cardbox"
+       ng-style="{ 'left': '0'}">
```
Comment (ernstkui) [inline app/components/omnibox/templates/omnibox.html]: This can just move to a scss file right?

File: app/index.html [hunk 4]
```diff
@@ -31,9 +31,10 @@
     <link rel="stylesheet" href="/styles/main.css">
     <!-- endbuild -->
 
+    <script>window.showomnibox = false;</script>
```
Comment (ernstkui) [inline app/index.html]: Leftover?

File: app/index.html [hunk 5]
```diff
@@ -31,9 +31,10 @@
...
   </head>
 
-  <body>
+  <body ng-class="{'dashboard': (context === 'dashboard'), 'map': (context === 'map')}">
```
Comment (ernstkui) [inline app/index.html]: What is this for?

File: app/lib/util-service.js [hunk 6]
```diff
@@ -625,16 +625,18 @@ angular.module('lizard-nxt')
    */
   this.getIconClass = function (str) {
```
Comment (ernstkui) [inline app/lib/util-service.js]: This only works for pumpstations, bridges and filters. What about all the other assets?

**Comment by ernstkui** [comment]
- [ ] Move inline styling to scss.

File: app/components/omnibox/directives/omnibox-directives.js (hunk 2 above)
Comment (gijs) [inline app/components/omnibox/directives/omnibox-directives.js]: Was intended for omnibox collapse/expand, not used anymore

File: app/components/omnibox/templates/omnibox.html (hunk 3 above)
Comment (gijs) [inline app/components/omnibox/templates/omnibox.html]: Done

File: app/index.html (hunk 4 above)
Comment (gijs) [inline app/index.html]: Yes, removed

File: app/lib/util-service.js (hunk 6 above)
Comment (gijs) [inline app/lib/util-service.js]: It falls back to just prefix `your_asset_name` with `lz lz-`, so everything in lizard-iconfont is supported. 
Font-awesome can also be supported but they have to be added here as exceptions.
I added comment/documentation for this function.

**Comment by ernstkui** [comment]
Done?

**Pull request merged**